src/main/java/org/broadinstitute/hellbender/engine/filters/ReadFilterLibrary.java

```diff
@@ -133,6 +133,22 @@ private ReadFilterLibrary(){ /*no instance*/ }
                     ! read.mateIsUnmapped() &&
                     read.mateIsReverseStrand() != read.isReverseStrand();}}
 
+    /**
+     * Keep only paired reads that are not near each other in a coordinate-sorted source of reads.
+     * (I.e., they're both mapped, but mapped to different contigs, or are mapped >= 1KB apart on the same contig.)
+     *
+     * <p>See MappedReadFilter for criteria defining an mapped read.</p>
+     */
+    @DocumentedFeature(groupName=HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY, summary = "Keep only paired reads with mates mapped >=1KB apart or on different contigs", extraDocs = MappedReadFilter.class)
+    public static class MateDistantReadFilter extends ReadFilter {
+        private static final long serialVersionUID = 1L;
+        public static final int DISTANT = 1000;
```

Comment:
I feel a little uncomfortable having a filter with a hard-coded value that's dependent on characteristics of the library in `ReadFilterLibrary`, which I think is supposed to be a collection of filters that might be generally useful across many GATK tools. Could you achieve the same result by applying `NotProperlyPairedReadFilter` and then within the downstream tool doing extra filtering on the reads (hopefully with the 1000 parameterized or learned from the data)?

Reply:
Failing that, is there any mechanism to let the user override this parameter? I don't see any other parameterized filters in this class but if there's a way to do that that would be good.

Reply:
I'll move this subclass out of the ReadFilterLibrary into a top-level class in the same package, and add a param.
I was trying to set this high enough that it would be library independent (because it was near the max allowed by the technology).  But it's certainly better to allow users to revise it.